privacy_partner_to_be_forgotten/readme/USAGE.md

```
@@ -0,0 +1,17 @@
+## To anonymize a partner record:
+
+NOTE:  **Make sure the user has permission** to anonymize partner records. Please check the Configuration file for details on how to grant permission.
+
+1. Go to the **Contacts App** in Odoo.
+2. Select the partner record you want to anonymize.
+3. Go to **Actions** → **Anonymize (GDPR)**.
+4. A confirmation wizard will appear. Press the **"Confirm"** button to proceed.
+
+Once you confirm, Odoo will anonymize the partner's record and all related child records, including:
+- Chatter messages and attachments
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)